providers/nodeReleasesDataProvider.tsx

```diff
@@ -0,0 +1,34 @@
+import { createContext } from 'react';
+import type { FC, PropsWithChildren } from 'react';
+import type {
+  NodeReleaseData,
+  NodeReleasesDataContext as NodeReleasesDataContextType,
+} from '../types';
+
+type NodeReleasesDataProviderProps = PropsWithChildren<{
+  nodeReleasesData: NodeReleaseData[];
+}>;
+
+export const NodeReleasesDataContext =
+  createContext<NodeReleasesDataContextType>(undefined as any);
+
+export const NodeReleasesDataProvider: FC<NodeReleasesDataProviderProps> = ({
+  children,
+  nodeReleasesData,
+}) => {
+  const current = nodeReleasesData.find(
+    nodeReleaseData => nodeReleaseData.status === 'Current'
+  );
+  const lts =
```

Comment:
Let's not do this. lts is either lts or undefined. We shouldn't fallback to the current release.

Reply:
It's from `useNodeData`
But... maybe it's better to keep it undefined, falling back to current is misleading... 🙈 
Thanks!